### PR TITLE
Update create-a-phone-system-auto-attendant-experiment.md

### PR DESCRIPTION
**Pull request by greganth1**
Update documentation concerning use of special characters in the text to speech greeting.

### CODE REVIEW

File: Teams/create-a-phone-system-auto-attendant-experiment.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Once you've set your auto attendant's general info, select **Next**.
 
 - If you select **Play an audio file** you can use the **Upload file** button to upload a recorded greeting message saved as audio in .WAV, .MP3, or .WMA format. The recording can be no larger than 5 MB.
 
-- If you select **Type a greeting message** the system will read the text that you type (up to 1000 characters) when the auto attendant answers a call.
+- If you select **Type a greeting message** the system will read the text that you type (up to 1000 characters) when the auto attendant answers a call.  Special characters such as "@" and "&" are not supported and their use can result in the greeting not playing. Use the full word such as "at" and "and" in the greeting text.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant-experiment.md:95]:
```suggestion
- If you select **Type a greeting message** the system will read the text that you type (up to 1,000 characters) when the auto attendant answers a call. Special characters such as "@" and "&" are not supported and their use could result in the greeting not playing. Use the full word such as "at" and "and" in the greeting text.
```

**Comment by scanum** [comment]
@DaniEASmith 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by DaniEASmith** [comment]
@scanum and @JohanFreelancer9

This doc also needs to be updated with these changes: https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness-pr/blob/live/Teams/create-a-phone-system-auto-attendant.md. The version edited in this PR is an experiment file from an A/B test that finished in December.

I copyedited the addition to read: If you select **Type a greeting message**, the system will read the text that you type (up to 1,000 characters) when the auto attendant answers a call. Special characters such as "@" and "&" aren't supported, and their use could result in the greeting not playing. Use the full word such as "at" and "and" in the greeting text.

My updates were just a few syntax edits.

**Comment by scanum** [comment]
@greganth1 could you please help submit the requested changes? Thanks.

**Comment by greganth1** [comment]
@scanum I'm not sure what I need to do.

**Pull request closed without merging**